### PR TITLE
update installation.md & favicon path

### PR DESCRIPTION
**Pull request by thenaturalist**
- fix installation requirements path to `requirements3.txt`
- fix favicon path in `amundsen_application/api/__init__.py` to refer correctly to static ressource

### CODE REVIEW

File: amundsen_application/api/__init__.py [hunk 1]
```diff
@@ -22,5 +22,5 @@ def healthcheck() -> Tuple[str, int]:
 
 def favicon() -> IO[bytes]:
     """ TODO: Design team should provide us with a default icon """
-    return send_from_directory(os.path.join(app.root_path, 'static/images'),
-                               'favicon.ico', mimetype='image/vnd.microsoft.icon')  # pragma: no cover
+    return send_from_directory(os.path.join(app.root_path, 'static/images/favicon.png'),
+                               mimetype='image/vnd.microsoft.icon')  # pragma: no cover
```
Comment (danwom) [inline amundsen_application/api/__init__.py]: I believe `send_from_directory` will return an error as the `directory` and `filename` parameters are required. That being said, this function related API are not used. 

The favicon is linked on line 14 of `static/templates/index.html`.
```
<link rel="shortcut icon" href="/static/images/favicon.png">
```

I would recommend deleting this API and function.

Comment (feng-tao) [reply amundsen_application/api/__init__.py]: @thenaturalist  @danwom I help to resolve some merge conflict and update the pr per Daniel's suggestion. Could you both take a look?

Comment (danwom) [reply amundsen_application/api/__init__.py]: You should also delete the route added on line 9.

Looks like whoever (me) added the favicon forgot to do some cleanup. 😄

**Comment by feng-tao** [comment]
@thenaturalist could you update the pr and fix the flake8? And we are good to go. Thanks for your contribution :)

**Comment by feng-tao** [comment]
thanks @thenaturalist for the pr.

**Pull request merged**